www/nginx/src/opnsense/service/templates/OPNsense/Nginx/http.conf

```diff
@@ -115,6 +115,10 @@ server {
     ssl_session_cache shared:SSL:50m;
     ssl_session_tickets off;
     ssl_prefer_server_ciphers on;
+{%     if server.ocsp_stapling is defined and server.ocsp_stapling == '1' %}
+    ssl_stapling on;
+    ssl_stapling_verify on;
```

Comment:
wrap that in an additional `if` and I prefer not to rely on the defaults.

Use an else block and only switch between on and off - in case of the semicolon as a last character the template engine will not strip the newline.

You can see an example after this diff in the file.